src/rules/no-special-in-object-members.js

```diff
@@ -0,0 +1,35 @@
+let specialSymbolsRegExp = /[!#%&()*+/=@^_]/;
+
+module.exports = {
+	meta: {
+		type: 'problem',
+		messages: {
+			errorMessage: `The object member have special symbol {{ identifier }}`
+		}
+	},
+	create(context) {
+		return {
+			VariableDeclaration(node) {
+				let objectProperties = node.declarations[0].init.properties;
+
+				objectProperties.forEach(property => {
+
+					let objectMemeber = property.key.value;
+					let specialSymbolInObjectMember = objectMemeber.match(specialSymbolsRegExp);
+
+
+					if (specialSymbolInObjectMember) {
+						let listOfSpecialSymbolsInThisObjectMember = specialSymbolInObjectMember.join('');
+						context.report({
+							node: node,
+							messageId: 'errorMessage',
+							data: {
+								identifier: listOfSpecialSymbolsInThisObjectMember
```

Comment:
What means `identifier`?

Reply:
Identifier is a data, inserted instead of `{{ ... }}` in error message

Reply:
Calling any identifier `identifier` for me is the same as calling variable `variable` and object property `property`

Reply:
I'm sorry. I thought `identifier` is reserved eslint `data` object member